### PR TITLE
Create hangman_new.py

### PR DESCRIPTION
**Pull request by ankitranjan9**

### CODE REVIEW

**Review by someshnarwade — CHANGES_REQUESTED** [review]
The formatting is very bad, that's why it's showing error while executing.
Please format it properly.

**Comment by ankitranjan9** [comment]
I have done the corrections and moved it to the other_solutions.
thank you

File: other_solutions/words.py [hunk 1]
```diff
@@ -0,0 +1 @@
...,    'actor',    'spark',    'dope',    'phrase',    'welsh',    'wall',    'whine',    'fickle',    'wrong',    'stamina',    'dazed',    'cramp',    'filet',    'foresee',    'seller',    'award',    'mare',    'uncover',    'drowning',    'ease',    'buttery',    'luxury',    'bigotry',    'muddy',    'photon',    'snow',    'oppress',    'blessed',    'call',    'stain',    'amber',    'rental',    'nominee',    'township',    'adhesive',    'lengthy',    'swarm',    'court',    'baguette',    'leper',    'vital',    'push',    'digger',    'setback',    'accused',    'taker',    'genie',    'reverse',    'fake',    'widowed',    'renewed',    'goodness',    'featured',    'curse',    'shocked',    'shove',    'marked',    'interact',    'mane',    'hawk',    'kidnap',    'noble',    'proton',    'effort',    'patriot',    'showcase',    'parish',    'mosaic',    'coil',    'aide',    'breeder',    'concoct',    'pathway',    'hearing',    'bayou',    'regimen',    'drain',    'bereft',    'matte',    'bill',    'medal',    'prickly',    'sarcasm',    'stuffy',    'allege',    'monopoly',    'lighter',    'repair',    'worship',    'vent',    'hybrid',    'buffet',    'lively']
```
Comment (someshnarwade) [inline other_solutions/words.py]: Instead of making a python file use a txt file and like 'word_list.txt'.

File: hangman_new.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import randomfrom words import word_list
```
Comment (someshnarwade) [inline hangman_new.py]: Delete this file.

File: other_solutions/README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
 ## Solutions by other users
 
-Add your solutions here with your github username as python file name.
+you can this program using your command line.
```
Comment (someshnarwade) [inline other_solutions/README.md:3]: Undo these changes

File: other_solutions/hang_man.py [hunk 4]
```diff
@@ -0,0 +1,146 @@
+import random
```
Comment (someshnarwade) [inline other_solutions/hang_man.py:1]: This works great.

**Review by someshnarwade — CHANGES_REQUESTED** [review]
I've added comments for the changes required. If you have questions, please ask.

**Comment by ankitranjan9** [comment]
I have deleted that file and changed words.py to words.txt. 
everything is done but not able to change the readme I have tried a lot.
please change it by yourself because from tomorrow my exams are going to start.
and please accept my PR this my 4th pr only.

**Review by someshnarwade — APPROVED** [review]
Approved

**Pull request merged**